data/h3_data_importer/Makefile

```diff
@@ -39,7 +40,7 @@ all:
 	fi
 	@echo "AWS S3 bucket access confirmed, proceeding with import..."
 	make clean
-	make -j 2 crop indicators
+	make indicators
```

Comment:
this should be still `-j 2 crop indicators`

Reply:
yes, you are right